Topics/starting-a-data-project.md

```diff
@@ -0,0 +1,254 @@
+# Starting a Data Project
+
+# What you will learn 
+In this guide, you'll learn to prepare yourself to start the project. 
+
+This will consist of learning how to look for sources that can help you solve the problem, building the project structure, build good documentation, secure your code, our data and more!
+
+In this guide we'll make heavy use of an awesome best practices collection from the [PLOS | Public Library of Science](https://www.plos.org/). 
+
+Do you know the happiness when you find a _giant diamond cluster in Minecraft_? 
+
+If yes, you need to know that you've just found a fat one:
+take your time to explore the [**Ten Rules Collection**](https://collections.plos.org/ten-simple-rules).
+
+
+## Prerequisites
+All the previous Purgatorio guides.
+
+## Time to complete
+10 minutes.
+
+# Index
+- [Collect Information](#Hunting-for-Information)
+- [Building a Knowledge Tree](#Building-a-Knowledge-Tree)
```

Comment:
Instead of knowledge tree -- have you considering suggesting to create [mind-maps](https://en.wikipedia.org/wiki/Mind_map) and/or [Rhizo-maps](http://spdrdng.com/posts/rhizomapping-rhizomaps-rhizomatic-learning-mindmapping-speed-reading-tip-17-take-notes-with-mindmaps-and-rhizomaps)?

This is a great practise when trying to formulate the problem statement -- totally helps dump (brain-dump) our thoughts into paper.

Reply:
done